### DISCUSSION TITLE
Consider requiring at least one screenshot

### DISCUSSION THREAD

**Discussion by alranel** [publiccodeyml/publiccode.yml#124] [General]
Maybe in a future version.

**Comment by AndreaTironi1** [comment]
A screenshot of a script Is awful, buy probably there will be free scripts.

Il Mer 14 Ago 2019, 17:07 Alessandro Ranellucci <notifications@github.com>
ha scritto:

**Comment by themightychris** [comment]
Not every project will have visual elements

**Comment by bfabio** [comment]
Moved to GitHub Discussions

<sub>[Why?](https://github.com/publiccodeyml/publiccode.yml/discussions/104)</sub>

**Comment by themightychris** [comment]
The "requirements" (as in things coded into the schema to flag files as invalid) should be as light as possible. Better for projects to start having a publiccode.yml early than to wait until they have more details and never get to it. The hard requirements should be oriented around a minimum viable project description -- probably just title+link+license. Beyond that, leave it to consumers of publiccode.yml files to express what additional fields are strongly recommended or even required for projects under their purview, but keep those opinions out of the common schema please. As a maintainer of a project that consumes publiccode.yml en masse, it doesn't help our efforts for project maintainers we're trying to encourage to adopt publiccode.yml to be getting confused by schema errors about fields we don't care about.

**Reply by ruphy** [reply]
I fully agree with you!
